recipes/greenlearning/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "greenlearning" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/NBoulle/greenlearning/archive/v{{ version }}.tar.gz"
+  sha256: 7d7d53d5d02d2123add2a58a04efebb6103bab1f2203fda2b287507fa50c5caa
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
If the only reason you're restricting Windows is a missing dependency, you can still keep it [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) and just ignore that. During installation, the solver will just inform the user of the missing dependency.

Reply:
The build fails on windows due to a missing dependency, so I can ignore this issue and keep it noarch right ?

Reply:
Yes, that's fine.

Reply:
It should be working now.

Reply:
Awesome, if you're now ready for review you can ping the team as outlined [here](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) or let me know if you have any questions.